pkgs/development/python-modules/databases/default.nix

```diff
@@ -4,41 +4,47 @@
 , sqlalchemy
 , aiocontextvars
 , isPy27
-, pytest
+, pytestCheckHook
+, pymysql
 , asyncpg
 , aiomysql
 , aiosqlite
 }:
 
 buildPythonPackage rec {
   pname = "databases";
-  version = "0.2.6";
+  version = "0.4.3";
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "encode";
     repo = pname;
     rev = version;
-    sha256 = "0cdb4vln4zdmqbbcj7711b81b2l64jg1miihqcg8gpi35v404h2q";
+    sha256 = "0aq88k7d9036cy6qvlfv9p2dxd6p6fic3j0az43gn6k1ardhdsgf";
   };
 
   propagatedBuildInputs = [
-    sqlalchemy
     aiocontextvars
+    sqlalchemy
   ];
 
   checkInputs = [
-    pytest
-    asyncpg
     aiomysql
     aiosqlite
+    asyncpg
+    pymysql
+    pytestCheckHook
   ];
 
-  # big chunk to tests depend on existing posgresql and mysql databases
-  # some tests are better than no tests
-  checkPhase = ''
-    pytest --ignore=tests/test_integration.py --ignore=tests/test_databases.py
-  '';
+  disabledTestPaths = [
+    # imports pymysql.util, which doesn't exist in pymysql>1.0
+    # https://github.com/encode/databases/issues/330
+    "tests/test_connection_options.py"
```

Comment:
should be fixed by this https://github.com/NixOS/nixpkgs/pull/122126

Reply:
databases just imports `aiomysql` and it was broken in nixpkgs because requirements were patched to allow PyMySQL 1.0 while aimysql wasn't patched, so that issue has nothing with databases, https://github.com/encode/databases/issues/330 should be probably closed.

Reply:
The patch you are pulling isn't merged in aiomysql either, so I consider this issue to still be relevant.

Reply:
Yes it's not pulled but if you install databases with pip then aiomysql will require PyMySQL < 1.0. That issue is caused by this patch https://github.com/NixOS/nixpkgs/blob/029d5b2fc2b9db1f401a89c1c242024fcbc36d47/pkgs/development/python-modules/aiomysql/default.nix#L37-L40